Add missing snapshots - part 1 (#1383)
tidymodels/recipes: tests/testthat/_snaps/date.md

      ! Name collision occurred. The following variable names already exist:
      * `Dan_year`

# errors on wrong values of features

    Code
      recipe(~ Dan + Stefan, examples) %>% step_date(all_predictors(), features = "yearly") %>%
        prep()
    Condition
      Error in `step_date()`:

>       x Possible values of `features` should include:
>       * "year", "doy", "mday", "week", "decimal", "semester", "quarter", "dow", or "month".
>       i Invalid values were: "yearly".

Reviewer: ```suggestion
      x Possible values of `features` are "year", "doy", "mday", "week", "decimal", 
         "semester", "quarter", "dow", or "month".
      i Invalid values were "yearly".
```

New snaps but not new erroring code—feel free to ignore—but the current message feels a bit overpunctuated to me. Something like this should be sufficiently richly-formatted to get the point across.

Author: done in [`e4ee075` (#1383)](https://github.com/tidymodels/recipes/pull/1383/commits/e4ee075669d8810e506f43ddcc4e35e55548cbc5)
